modules/apps/questions/questions-web/test/js/utils/utils.es.js

```diff
@@ -21,5 +21,34 @@ describe('utils', () => {
 				'lorem ipsum dolor'
 			);
 		});
+
+		it('return a text with a hyphen and a space', () => {
+			expect(Utils.slugToText('Lorem--ipsum-dolor')).toEqual(
+				'lorem-ipsum dolor'
+			);
+			expect(Utils.slugToText('Lorem--ipsum--dolor')).toEqual(
```

Comment:
make a new it block or replace the text to be more generic and fit all the cases that are beeing tested